min_compute.yml

```diff
@@ -22,12 +22,12 @@ compute_spec:
 
     gpu:
       required: True                       # Does the application require a GPU?
-      min_vram: 20                         # Minimum GPU VRAM (GB)
-      recommended_vram: 24                 # Recommended GPU VRAM (GB)
+      min_vram: 80                         # Minimum GPU VRAM (GB)
```

Comment:
Shouldn't this be 60?

Reply:
> Shouldn't this be 60?

I was not able to fit Llama3 70B AWQ without `enforce_eager=True` into 60GB. Based on my experiments 75 should work

Reply:
That's a good catch @p-ferreira , we don't need 80 as minimum.
Based on this PR 62GB works well on 2xA40 for me: https://github.com/macrocosm-os/prompting/pull/235#discussion_r1608304923